src/pages/Home.tsx

```diff
@@ -1,6 +1,30 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Link } from 'react-router-dom';
 
+function toHexString (byteArray: Uint8Array): string {
+  return Array.from(byteArray, (byte: number) => {
+    return ('0' + (byte & 0xff).toString(16)).slice(-2);
+  }).join('');
+}
```

Comment:
Super cryptic! Should we include this in sesamoid? I think `byte & 0xff` is not needed because we have unsigned bytes.

Reply:
>  Should we include this in sesamoid?

Good idea, we can use this crate here to return the public and private keys as hex: https://docs.rs/hex/0.4.2/hex/fn.encode.html

> I think byte & 0xff is not needed because we have unsigned bytes.

Aha!

Reply:
done!